phpstan.neon.dist

```diff
@@ -30,12 +30,18 @@ parameters:
   excludePaths:
     - admin-dev/filemanager
     - admin-dev/themes
+    - classes/lang/KeysReference
   ignoreErrors:
     - '#^Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition\:\:#'
     - '#^Unsafe usage of new static\(\)\.$#'
     - '#^Constructor of class PrestaShopBundle\\Service\\DataProvider\\Marketplace\\ApiClient has an unused parameter \$toolsAdapter\.$#'
     - '#constructor expects Symfony\\Contracts\\Translation\\TranslatorInterface\|null, Symfony\\Component\\Translation\\TranslatorInterface given\.$#'
     - '#^Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface\:\:dispatch\(\) invoked with 2 parameters, 1 required\.$#'
+    ## Relative to global module variable used for translation
+    - '#^Undefined variable: \$_MODULE$#'
+    ## Smarty functions
+    - '#^Function smartyRegisterFunction not found\.$#'
```

Comment:
Why? Maybe the bootstrap has to be improved?

Reply:
I try but we can't include `config/smartyadmin.config.inc.php` because the file is a mix of functions and code (with some use of `Configuration`) which can't be autoloaded as `Configuration` can't be used.

Reply:
it sucks